src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -939,17 +939,50 @@
 helps['openshift create'] = """
 type: command
 short-summary: Create a new managed OpenShift cluster.
+parameters:
+  - name: --compute-vm-size -s
+    type: string
+    short-summary: Size of Virtual Machines to create as OpenShift nodes.
+  - name: --compute-count -c
+    type: int
+    short-summary: Number of nodes in the OpenShift node pool.
+  - name: --aad-client-app-id
+    type: string
+    short-summary: The ID of an Azure Active Directory client application. If not specified, a new Azure Active Directory client is created.
+  - name: --aad-client-app-secret
+    type: string
+    short-summary: The secret of an Azure Active Directory client application.
+  - name: --aad-tenant-id
+    type: string
+    short-summary: The ID of an Azure Active Directory tenant.
+  - name: --vnet-peer
+    type: string
+    short-summary: The ID or the name of a subnet in an existing VNet into which to peer the cluster.
+  - name: --vnet-prefix
+    type: string
+    short-summary: The CIDR used on the VNet into which to deploy the cluster.
+  - name: --subnet-prefix
+    type: string
+    short-summary: The CIDR used on the Subnet into which to deploy the cluster.
+  - name: --customer-admin-group-id
+    type: string
+    short-summary: The Object ID of an Azure Active Directory Group that memberships will get synced into the OpenShift group "osa-customer-admins". If not specified, no cluster admin access will be granted.
+  - name: --workspace-id
+    type: string
+    short-summary: The resource id of an existing Log Analytics Workspace to use for storing monitoring data.
```

Comment:
Does it support name?

Reply:
Hi @qwordy this is a revert to the code that was there before. We had a numerous discussions about this name in the past. Also we are sunsetting the ARO v3 service and do not want to introduce any more changes.